.env.production

```diff
@@ -1,7 +1,7 @@
 APP_NAME=Agripoll
```

Comment:
This is what you would change to Buzza Agripoll. The other olternative in the config, is only taken if this VARIABLE is not set. Other wise the AppName is still Agripoll

Reply:
Check my comment below.